Tests/KituraNetTests/ConnectionLimitTests.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+import Dispatch
+import NIO
+import XCTest
+import KituraNet
+import NIOHTTP1
+import NIOWebSocket
+import LoggerAPI
+
+class ConnectionLimitTests: KituraNetTest {
+    static var allTests: [(String, (ConnectionLimitTests) -> () throws -> Void)] {
+        return [
+            ("testConnectionLimit", testConnectionLimit),
+        ]
+    }
+    
+    override func setUp() {
+        doSetUp()
+    }
+
+    override func tearDown() {
+        doTearDown()
+    }
+    private func sendRequest(request: HTTPRequestHead, on channel: Channel) {
+        channel.write(NIOAny(HTTPClientRequestPart.head(request)), promise: nil)
+        try! channel.writeAndFlush(NIOAny(HTTPClientRequestPart.end(nil))).wait()
+    }
+    
+    func establishConnection(expectation: XCTestExpectation, responseHandler: HTTPResponseHandler) {
+        var channel: Channel
+        let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        let bootstrap = ClientBootstrap(group: group)
+            .channelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
+            .channelInitializer { channel in
+                channel.pipeline.addHTTPClientHandlers().flatMap {_ in
+                    channel.pipeline.addHandler(responseHandler)
+                }
+        }
+        do {
+            try channel = bootstrap.connect(host: "localhost", port: self.port).wait()
+            let request = HTTPRequestHead(version: HTTPVersion(major: 1, minor: 1), method: .GET, uri: "/")
+            self.sendRequest(request: request, on: channel)
+        } catch _ {
+            XCTFail("Connection is not established.")
+        }
+    }
+
+func testConnectionLimit() {
```

Comment:
Can you fix the indentation for the rest of this file? It's difficult to tell whether (for example) this function and the classes defined below are within the scope of `ConnectionLimitTests` or not.